core/src/main/java/discord4j/core/object/entity/Guild.java

```diff
@@ -1370,6 +1371,16 @@ public OptionalInt getMaxMembers() {
                 });
     }
 
+    /**
+     * Requests to retrieve the voice connection that the client is propagating in this guild.
```

Comment:
I think we don't need to use the wording "requests to retrieve" here. Maybe just modify the summary from `VoiceConnectionRegistry#getVoiceConnection(Snowflake)`.